docs/installation.md

```diff
@@ -0,0 +1,103 @@
+# Installation
+
+## Deploy the Gateway
+
+You can deploy NGINX Kubernetes Gateway on an existing Kubernetes 1.16+ cluster. The following instructions walk through the steps for deploying on a [kind](https://kind.sigs.k8s.io/) cluster.
```

Comment:
For sake of completeness, we used to mention `kubectl`, I think a note about it with `kind` is reasonable. `[kubectl](https://kubernetes.io/docs/tasks/tools/)`

Reply:
Where is the kubectl requirement mentioned?

Reply:
@f5yacobucci in the prerequisites section